From 3dbe4e551c7fc1fa1bf76d6efa70a0ad61c630ac Mon Sep 17 00:00:00 2001
From: "Carsten Haitzler (Rasterman)" <raster@rasterman.com>
Date: Sat, 22 May 2021 14:24:42 +0100
Subject: [PATCH 11/13] alpm - silence warning for gool to ptr conversion with
 a cast

too many warnings makse us miss the real problems... and i have found
a fair few already. let's ne more silent
---
 backends/alpm/pk-alpm-sync.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/backends/alpm/pk-alpm-sync.c b/backends/alpm/pk-alpm-sync.c
index abc6fb1d5..8c7ded998 100644
--- a/backends/alpm/pk-alpm-sync.c
+++ b/backends/alpm/pk-alpm-sync.c
@@ -264,7 +264,7 @@ pk_backend_update_packages (PkBackend *self,
 			    PkBitfield transaction_flags,
 			    gchar **package_ids)
 {
-	pk_alpm_run (job, PK_STATUS_ENUM_SETUP, pk_backend_sync_thread, TRUE);
+	pk_alpm_run (job, PK_STATUS_ENUM_SETUP, pk_backend_sync_thread, (void *)TRUE);
 }
 
 void
-- 
2.31.1

